src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/Verbs.cs

```diff
@@ -66,6 +66,82 @@ internal partial class AddVerbParser
                         return 0;
                     });
                 });
+                AddCmd.Command("client-cert", ClientCertCmd =>
```

Comment:
I thought I previously posted this comment, but now I can't find it, so sorry if this is a duplicate.

Anyway, the top of this file has a comment saying it's generated from commands.xml and this file shouldn't be hand edited, but I don't see commands.xml in the diff. Did you see the comment at the top of the file? If not, do you have a suggestion how we can make it more clear?

cc @rrelyea

Reply:
Yeah missed that header :) Will modify this in right way later today. The way how this works is not intuitive. I would suggest you to rename automatically generated files by adding postfix (*.generated.cs for example)

Reply:
Btw have you instruction that describe whole commands generation process?

Reply:
The comment at the top of the file says:

> // Do not manually edit this autogenerated file:
> // instead modify the neighboring .tt file (text template) and/or NuGet.CommandLine.Xplat\Commands\Commands.xml (data file),
> // then re-execute the text template via "run custom tool" on VS context menu for .tt file, or via dotnet-t4 global tool.

Is that sufficient?

Reply:
Done